.github/workflows/build-vcpkg.yml

```diff
@@ -31,43 +31,58 @@ jobs:
   build:
     strategy:
       matrix:
-        os: ["ubuntu-20.04", "ubuntu-22.04", "windows-2019", "windows-2022", "macos-11", "macos-12"]
+        os:
+          [
+            "ubuntu-18.04",
```

Comment:
style/question/picky: I think short lists are normally(often) short formed to the previous from [ 1, 2, 3 ],
but long lists are usually listed one per line, i.e.:
```
            os:
            - "ubuntu-20.04"
            - "ubuntu-22.04"
            - "windows-2019"
            - "windows-2022"
            - "macos-11"
            - "macos-12"
```

Reply:
Its the yaml editor pretty print doing this, I can revert to one line if you would prefer?

Reply:
no big deal, every reference I see states that the [ 1, 2, 3 ] is the single line form, with the multiline as above.